pkg/skaffold/initializer/build/cli.go

```diff
@@ -66,12 +66,12 @@ func (c *cliBuildInitializer) processCliArtifacts() error {
 	if err != nil {
 		return err
 	}
-	c.builderImagePairs = pairs
+	c.artifactInfos = pairs
 	return nil
 }
 
-func processCliArtifacts(cliArtifacts []string) ([]BuilderImagePair, error) {
-	var pairs []BuilderImagePair
+func processCliArtifacts(cliArtifacts []string) ([]ArtifactInfo, error) {
+	var pairs []ArtifactInfo
```

Comment:
Any plans to rename the variable names?

Reply:
oh whoops, totally missed that 😅 I'll make another pass through and try to fix any of those I notice